File: qualifier-week-4/graded-assignments/GrPA8 - Graded.lua
local total = 0.0

while true do
    io.write("Enter the price of the item: ")
    local price = tonumber(io.read())

    total = total + price

    io.write("Do you want to add another item? (y/n): ")
    local choice = io.read()

    if choice ~= 'y' and choice ~= 'Y' then
        break
    end
end

print(string.format("Total cost: %.2f", total))

File: qualifier-week-4/graded-assignments/GrPA8 - Graded.go
package main

import "fmt"

func main() {
	var total, price float64
	var choice string

	for {
		// Prompt the user for the item price
		fmt.Scan(&price)

		// Add the price to the total cost
		total += price

		fmt.Print("Do you want to add another item? (y/n): ")
		fmt.Scan(&choice)

		if choice != "y" && choice != "Y" {
			break
		}
	}

	// Print the total cost
	fmt.Printf("%.2f\n", total)
}
